api/builtins_qlik/Gomplate.go

```diff
@@ -60,19 +61,24 @@ func (p *GomplatePlugin) Transform(m resmap.ResMap) error {
 		}
 	}
 
-	var ejsonPrivateKeyPath string
+	var ejsonKey string
```

Comment:
This change was just for fixing the following pesky and unnecessary error log on every transform:
```
error executing stat on ejson private key file: %!s(<nil>), error: stat %!s(<nil>): no such file or directory
```